tick/optim/model/src/model_labels_features.cpp

```diff
@@ -1,21 +1,39 @@
-// License: BSD 3 clause
-
-//
-// Created by Stéphane GAIFFAS on 06/12/2015.
-//
-
 #include "model_labels_features.h"
 
+// License: BSD 3 clause
```

Comment:
It doesn't worth pushing a new commit for this, but usually, in cpp files, licence is a the very beginning of the file